libs/ui/design-system/src/lib/components/ShowMoreText/ShowMoreText.tsx

```diff
@@ -0,0 +1,84 @@
+import classNames from "classnames";
+import { CSSProperties, useCallback, useEffect, useRef, useState } from "react";
+
+import "./ShowMoreText.scss";
+
+const CLASS_NAME = "amp-show-more-text";
+
+export type Props = {
+  gapAfterText?: number;
+  lineClamp?: number;
+  children: React.ReactNode;
+};
+
+export const ShowMoreText = ({
+  children,
+  gapAfterText = 20,
```

Comment:
this should be set by design and not dynamic. it'll be probably one design for design system